File-Formats/PDF/Working-with-DigitalSignature.md

```diff
@@ -6092,7 +5416,7 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Adding multiple signatures to a PDF document
 
-The following code example illustrates how to add multiple signatures to a PDF document without invalidating the previous signature.  
+The following code example illustrates how to add multiple signatures to a PDF document using [PdfLoadedSignatureField](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Parsing.PdfLoadedSignatureField.html) class without invalidating the previous signature.  
```

Comment:
The following code example illustrates how to add multiple signatures to a PDF document without invalidating the previous signature.

Reply:
Changed the content